arbitrum-docs/run-arbitrum-node/data-availability-committees/02-deploy-das.md

```diff
@@ -274,10 +275,9 @@ To activate the "archive mode" in your DAS, set the parameter `discard-after-tim
 
 ```bash
 --data-availability.s3-storage.discard-after-timeout=false
---data-availability.local-db-storage.discard-after-timeout=false
 ```
 
-Note that `local-file-storage` and `ipfs-storage` don't discard data after expiring, so the option `discard-after-timeout` is not available.
+Note that `local-file-storage` doesn't discard data after expiring, so the option `discard-after-timeout` is not available.
```

Comment:
This is no longer true, there is an `--data-availability.local-file-storage.enable-expiry` option now and `--data-availability.local-file-storage.max-retention`

Reply:
I'm assuming then it doesn't discard data by default, but can discard data if `enable-expiry` is set to true?

Also, `max-retention` seems to be related more to store requests received, rather than discarding (expiring) data it already contains, is that correct? (just figuring out if I need to include it in this specific paragraph).

Reply:
Yes this is a correct understanding.